core/src/main/java/org/jdbi/v3/core/statement/SqlLoggerUtil.java

```diff
@@ -23,18 +25,20 @@ private SqlLoggerUtil() {
     }
 
     static <T> T wrap(SqlLoggable<T> r, StatementContext ctx, SqlLogger logger) throws SQLException {
+        Clock clock = ctx.getConfig(Time.class).getClock();
+
         try {
-            ctx.setExecutionMoment(Instant.now());
+            ctx.setExecutionMoment(Instant.now(clock));
```

Comment:
well this is the preferred way to do it, because Clock only offers a method to get an Instant, but the other 9 Temporal types all have a `now` method with an overload that takes a Clock, so it's more consistent that way.

Reply:
ok